android-spaces-real-time-app/app/src/main/java/com/mux/realtimevideoexample/MainActivity.java

```diff
@@ -63,7 +65,7 @@ public void onParticipantTrackUnsubscribed(Space space, Participant participant,
 
         @Override
         public void onError(Space space, MuxError muxError) {
-            Toast.makeText(MainActivity.this, "Error! "+muxError.toString(), Toast.LENGTH_LONG).show();
+            Toast.makeText(MainActivity.this, "Error! "+muxError.toString(), Toast.LENGTH_SHORT).show();
```

Comment:
Can we keep these as LONG? I already have enough problem spotting the Long ones 😅

Reply:
Agreed, and changed.